src/components/CategorySelector.tsx

```diff
@@ -56,16 +56,19 @@ interface CategorySelectorProps {
   selectedCategories: PublicationInstanceType[];
   onCategoryClick?: (category: PublicationInstanceType) => void;
   disabledCategories?: DisabledCategory[];
+  enableNviHighlightning: boolean;
```

Comment:
Mtp de andre kommentarene blir denne noe sånt i stedet tror jeg. Navnet klinger dårlig, men det jog kommer på i farta :D 
```suggestion
  canSelectAllNviCategories: boolean;
```